randovania/gui/main_window.py

```diff
@@ -292,9 +347,16 @@ def showEvent(self, event: QtGui.QShowEvent):
         self.InitPostShowSignal.emit()
 
     # Per-Game elements
-    def refresh_game_list(self):
-        for game, logo in self._play_game_logos.items():
-            logo.setVisible(game.data.development_state.can_view())
+    def refresh_game_list(self) -> None:
+        for game, game_elements in self._play_game_elements.items():
+            game_elements.tile.setVisible(game.data.development_state.can_view())
+            if game_elements.tile.isVisible() and isinstance(game_elements.multi_banner, QtWidgets.QLabel):
+                game_elements.multi_banner.setVisible(
+                    game.data.defaults_available_in_game_sessions and self._options.show_multiworld_banner
+                )
+                game_elements.multi_icon.setVisible(
```

Comment:
The mypy thing here tells that `None` has no `setVisible` method and the code will fail in that case. If `QLabel | None` is really required (and not only `QLabel`), do something like
```
label = game_elements.multi_icon
assert isinstance(label, QLabel)
label.setVisible(....
```

Reply:
You did it differently but it's also fine. Just add the `isinstance` check also for `multi_icon` then mypy is quiet and everything good to go

Reply:
I think the beast is tamed. Hard to tell. Do you think the `isinstance` is still necessary?

Reply:
No not necessary. As you now removed the `None` type, there is no need to check the instance anymore.